homeassistant/components/shelly/entity.py

```diff
@@ -20,6 +22,25 @@ def temperature_unit(block_info: dict) -> str:
     return TEMP_CELSIUS
 
 
+def shelly_rest_parser(self, path: str):
+    """Parser for REST path from device status."""
+
+    if "/" not in path:
+        _attribute_value = self.wrapper.device.status[path]
+    else:
+        _attribute_value = self.wrapper.device.status[path.split("/")[0]][
+            path.split("/")[1]
+        ]
+    if self.description.device_class == DEVICE_CLASS_TIMESTAMP:
```

Comment:
This is weird. This parsing should be passed as a parameter to `RestAttributeDescription`, just like we do for other attributes.

Reply:
self.description is an object of type RestAttributeDescription.